addons/website/static/src/components/navbar/navbar.scss

```diff
@@ -1,5 +1,10 @@
 .o_menu_systray.o_website_systray {
-    @if not $o-navbar-use-pills {
+    @if $o-navbar-use-pills {
+        --NavBar-entry-margin-right: #{$o-navbar-entry-padding-h};
+    } @else {
+        --NavBar-entry-padding-left: #{$o-navbar-entry-padding-h};
+        --NavBar-entry-padding-right: #{$o-navbar-entry-padding-h};
+
```

Comment:
Is this related to this or not: I still have the same bug in enterprise as I mentioned before. There is a gap between buttons where the cursor is like "you are not clicking on anything" but if you click, you are actually triggering the button (e.g. between publish button and mobile button).

Reply:
Fixed by sri